mlrun/feature_store/steps.py

```diff
@@ -383,6 +470,18 @@ def __init__(
         super().__init__(**kwargs)
         self.timestamp_col = timestamp_col
         self.parts = parts
+        self.spark_dict_str = {
```

Comment:
I think this can have more meaningful name, like `x_to_spark_date_format` where `x` is the type of string that you cast there. For example, if in feature store these are the terms then you can name it `fstore_date_format_to_spark_date_format`.